debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresPartition.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+import io.debezium.connector.common.Partition;
+import io.debezium.util.Collect;
+
+public class PostgresPartition implements Partition {
+    private static final String SERVER_PARTITION_KEY = "server";
```

Comment:
The only existing connector that doesn't use it is MongoDB: https://github.com/debezium/debezium/blob/5b76bf1b713f27a9f2bc70b9ea1484a441693ffa/debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/SourceInfo.java#L71-L72

Which I believe isn't intended, it could use it as well.

The question is, whether there's a plan to extend Debezium to support "serverless" sources, e.g. S3? In this case, it would make sense to keep the "server" out of the API.

I'd leave it as-is for now. Currently, each `OffsetContext` defines its own partition keys, and this change follows the existing pattern. I believe there will be a better way to generalize this logic (e.g. how it's done with the `Relational*` classes) once we more code changes in place.

Reply:
Ok, makes sense. Thx!